README.md

```diff
@@ -407,6 +407,18 @@ The `@StateObject` documentation reads:
 
 `@Instantiated`, `@Forwarded`, or `@Received` objects may be decorated with [`@ObservedObject`](https://developer.apple.com/documentation/swiftui/ObservedObject). Note that `@Instantiated` objects declared on a `View` will be re-initialized when the view is re-initialized. You can find a deep dive on SwiftUI view lifecycles [here](https://www.donnywals.com/understanding-how-and-when-swiftui-decides-to-redraw-views/).
 
+#### Dependency cycles
+
+In a manual DI system it is easy to instantiate an object that can instantiate another object that can then instantiate the first object again. Instantiable objects in SafeDI, however, are not able to generate this kind of circular dependency. Instantiable objects can achieve this kind of circular dependency manually: one object in the cycle must call other cycle objects‘ memberwise initializers directly, rather than utilizing generated `Instantiator`s or `ForwardingInstantiator`s.
+
+#### Inheritance
```

Comment:
while I'd love to tell people "just use composition"... SafeDI advertises itself as architecture independent. So instead we explain how to utilize SafeDI with inheritance trees.